pytorch_lightning/core/lightning.py

```diff
@@ -1685,6 +1685,10 @@ def summarize(self, mode: Optional[str] = "top", max_depth: Optional[int] = None
         """
         Summarize this LightningModule.
 
+            .. deprecated:: v1.5
+                This method was deprecated in v1.5 in favor of `pytorch_lightning.utilities.model_summary.summarize`
+                and will be removed in v1.7.
+
```

Comment:
Could you un-indent this so it gets properly rendered in the html output? thx :)